src/components/ElectionComponent.tsx

```diff
@@ -1,19 +1,146 @@
+                expandedElectionId={expandedElectionId}
+                onExpandElectionRow={onExpandElectionRow}
+              />
+            ))}
+          </TableBody>
+        </Table>
+      </TableContainer>
+      <CreateElectionComponent />
+    </Container>
+  );
+}
+
+const useRowStyles = makeStyles({
+  root: {
+    '& > *': {
+      borderBottom: 'unset',
+    },
+  },
+});
+
+type ElectionRowProps = {
+  election: Election;
+  expandedElectionId: number;
+  onExpandElectionRow: (expandedElectionId: number) => void;
+};
+
+function ElectionRow({ election, expandedElectionId, onExpandElectionRow }: ElectionRowProps) {
+  const classes = useRowStyles();
+
   return (
     <>
-      <h1>Elections!</h1>
-      <ul>
-        {props.elections?.map(election => {
-          return <li key={election.id}>{election.name}</li>;
-        })}
-      </ul>
+      <TableRow className={classes.root}>
+        <TableCell>
+          <IconButton aria-label="expand row" size="medium" onClick={() => onExpandElectionRow(election.id)}>
+            {election.id === expandedElectionId ? <KeyboardArrowUp /> : <KeyboardArrowDown />}
+          </IconButton>
+        </TableCell>
+        <TableCell component="th" scope="row">
+          {election.name}
+        </TableCell>
+        <TableCell>{election.year}</TableCell>
+        <TableCell>{election.description}</TableCell>
+      </TableRow>
+      <TableRow>
+        <TableCell style={{ paddingBottom: 0, paddingTop: 0 }} colSpan={4}>
+          <Collapse in={election.id === expandedElectionId} timeout="auto" unmountOnExit>
+            <Box margin={1}>
+              <Typography variant="h6" gutterBottom component="div">
+                Results
+              </Typography>
+              <Table size="small">
+                <TableHead>
+                  <TableRow>
+                    <TableCell>Question</TableCell>
+                    <TableCell>Yes</TableCell>
+                    <TableCell>No</TableCell>
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {election.questions?.map(question => (
+                    <TableRow key={question.id}>
+                      <TableCell component="th" scope="row">
+                        {question.content}
+                      </TableCell>
+                      <TableCell>{question.yes}</TableCell>
+                      <TableCell>{question.no}</TableCell>
+                    </TableRow>
+                  ))}
+                </TableBody>
+              </Table>
+            </Box>
+          </Collapse>
+        </TableCell>
+      </TableRow>
     </>
   );
 }
+
+function CreateElectionComponent() {
+  // const [newElection, setNewElection] = useState<ElectionForm>({
+  //   name: '',
+  //   year: '2020',
+  //   description: '',
+  //   questions: [],
+  // });
+
+  // const change = (event: SyntheticEvent) => {
+  //   console.log(event.target.value);
+  // };
```

Comment:
delete?

Reply:
Working on this right now, just wanted it commented out of the PR